keyboards/cannonkeys/iron165/keymaps/via/rules.mk

```
@@ -2,4 +2,4 @@
 
 RAW_ENABLE = yes
 DYNAMIC_KEYMAP_ENABLE = yes
-
+VIA_ENABLE = yes
```

In theory, only need this: 
```suggestion
VIA_ENABLE = yes
```

---

Just tested, works fine!